src/worker.js

```diff
@@ -8,11 +8,16 @@ import BatchManager from './utils/BatchManager';
 
 let closing = false;
 
+const healthCheckEndpoint = '/';
```

Comment:
I think overloading the root URL is not a good idea; if we want this, `/health-check` or something similar would be better.